internal/nexodus/ctlpeers.go

```diff
@@ -0,0 +1,31 @@
+package nexodus
+
+import (
+	"encoding/json"
+	"fmt"
+	"runtime"
+)
+
+func (ac *NexdCtl) ListPeers(_ string, result *string) error {
+	var iface string
+
+	if runtime.GOOS == Darwin.String() {
+		iface = darwinIface
+	} else {
+		iface = wgIface
+	}
```

Comment:
It's probably worth using `defaultTunnelDev()` which implements this same logic, but also handles the `proxy` mode.